doc/sphinx-guides/source/installation/config.rst

```diff
@@ -594,7 +594,7 @@ You may also want to look at samples at https://github.com/shlake/LibraDataHomep
 A simpler option to brand and customize your installation is to utilize the Dataverse collection theme, which each Dataverse collection has, that allows you to change colors, add a logo, tagline or website link to the Dataverse collection header section of the page. Those options are outlined in the :doc:`/user/dataverse-management` section of the User Guide.
 
 Custom Homepage
-++++++++++++++++
++++++++++++++++
```

Comment:
FWIW: I think my rst plugin is catching these now. I just added the two entries at the bottom.